### PR TITLE
Add capacity to click on the notification

### PR DESCRIPTION
**Pull request by Quentin-M**

### CODE REVIEW

File: notification_center.py [hunk 1]
```diff
@@ -37,15 +38,16 @@ def notify(data, buffer, date, tags, displayed, highlight, prefix, message):
 
 	# passing `None` or `''` still plays the default sound so we pass a lambda instead
 	sound = weechat.config_get_plugin('sound_name') if weechat.config_get_plugin('sound') == 'on' else lambda:_
+	activate_identifier = weechat.config_get_plugin('activate_identifier')
 	if weechat.config_get_plugin('show_highlights') == 'on' and int(highlight):
 		channel = weechat.buffer_get_string(buffer, 'localvar_channel')
-		if weechat.config_get_plugin('show_message_text') == 'on':
-			Notifier.notify(message, title='%s %s' % (prefix, channel), sound=sound, appIcon=WEECHAT_ICON)
+		if weechat.config_get_plugin('show_message_itext') == 'on':
```
Comment (sindresorhus) [inline notification_center.py]: Why?

File: notification_center.py [hunk 2]
```diff
@@ -21,6 +21,7 @@
 	'show_message_text': 'on',
 	'sound': 'off',
 	'sound_name': 'Pong',
+	'activate_identifier': 'com.apple.Terminal',
```
Comment (sindresorhus) [inline notification_center.py]: I would go with `activate_bundle_id`.

File: notification_center.py (hunk 1 above)
Comment (Quentin-M) [inline notification_center.py]: My bad, fixing. Nice catch.

**Comment by sindresorhus** [comment]
Can you add it to the readme too? Would be useful to also include the bundle ID of iTerm, as many use it.

**Comment by sindresorhus** [comment]
Good idea! :)

**Comment by Quentin-M** [comment]
Updated !

I'd like to do more actually: the click would spawn your term, select the right tab/pane and then open the right buffer. But I don't think that's actually possible and that PR is already a nice improvement imho.

**Comment by sindresorhus** [comment]
> the click would spawn your term, select the right tab/pane and then open the right buffer

Should be possible to use AppleScript to find the correct tab by looking at the title and then select it. Selecting the right buffer would be more difficult, but probably possible.

-

Another improvement I just thought is auto-detecting the terminal using the `$TERM_PROGRAM` environment variable. http://stackoverflow.com/questions/7052683/how-can-i-identify-the-current-terminal-emulator-from-a-bash-script

**Comment by sindresorhus** [comment]
Do you want to work on any of these or should I do a release right away?

**Pull request merged**